### ISSUE TITLE
EquipmentManager_UnpackLocation returns wrong bag id for bank bags

### ISSUE THREAD

**Issue by mbattersby** [Stanzilla/WoWUIBugs#343]
EquipmentManager_UnpackLocation hasn't been updated for the existence of the Reagent Bag and returns a bag ID one too low for bank bags.

The main cause is `ITEM_INVENTORY_BANK_BAG_OFFSET = 4` in FrameXML/Constants.lua should now be 5. Setting to 5 appears to resolve it. Since this is the only place this constant is used EquipmentManager_UnpackLocation could use `NUM_TOTAL_EQUIPPED_BAG_SLOTS` instead.

This causes a Lua error in the default UI using the EquipmentFlyout buttons while the bank is open.

Reproduce:

1. Put an item for an equipment slot (e.g. glove) in a bank bag.
2. Make sure there is no item in (bag-1, slot).
3. Open the bank and the PaperDoll frame.
4. Click on the PaperDoll frame Equipment Manager tab.
5. Click on the EquipmentFlyout button next to that equipment slot.

Error from EquipmentFlyout:

```
Message: Interface/FrameXML/EquipmentManager.lua:300: Usage: GetItemInfo(itemID|"name"|"itemlink")
Time: Sat Nov 12 17:17:39 2022
Count: 4
Stack: Interface/FrameXML/EquipmentManager.lua:300: Usage: GetItemInfo(itemID|"name"|"itemlink")
[string "=[C]"]: in function `GetItemInfo'
[string "@Interface/FrameXML/EquipmentManager.lua"]:300: in function `EquipmentManager_GetItemInfoByLocation'
[string "@Interface/FrameXML/EquipmentFlyout.lua"]:452: in function `EquipmentFlyout_DisplayButton'
[string "@Interface/FrameXML/EquipmentFlyout.lua"]:315: in function `EquipmentFlyout_UpdateItems'
[string "@Interface/FrameXML/EquipmentFlyout.lua"]:231: in function `EquipmentFlyout_Show'
[string "@Interface/FrameXML/EquipmentFlyout.lua"]:629: in function <Interface/FrameXML/EquipmentFlyout.lua:621>

Locals: (*temporary) = nil
```

**Comment by mbattersby** [comment]
Present in both beta build 46619 and retail build 46597.

**Comment by mbattersby** [comment]
This is fixed in 10.0.5 PTR.
